dev/generate_tokens/base_tokens_map.js

```diff
@@ -21,176 +21,5 @@ module.exports = {
     ),
     [ChainId.POLYGON]: new Token(ChainId.POLYGON, '0x831753dd7087cac61ab5644b308642cc1c33dc13', 18, 'QUICK', 'Quick'),
     [ChainId.AURORA]: new Token(ChainId.AURORA, '0x0b20972b45ffb8e5d4d37af4024e1bf0b03f15ae', 18, 'WETH', 'Ethereum')
-  },
-
-  USDT: {
-    [ChainId.FUJI]: new Token(ChainId.FUJI, ZERO_ADDRESS, 6, 'USDT', 'Tether USD'),
-    [ChainId.AVALANCHE]: new Token(
-      ChainId.AVALANCHE,
-      '0xde3A24028580884448a5397872046a019649b084',
-      6,
-      'USDT',
-      'Tether USD'
-    ),
-    [ChainId.POLYGON]: new Token(
-      ChainId.POLYGON,
-      '0xc2132d05d31c914a87c6611c10748aeb04b58e8f',
-      6,
-      'USDT',
-      'Tether USD'
-    ),
-    [ChainId.AURORA]: new Token(ChainId.AURORA, '0x4988a896b1227218e4a686fde5eabdcabd91571f', 6, 'USDT', 'Tether USD')
```

Comment:
For this file, in order to allow the `tokens` repo to be the single source of truth, what do you think about only removing the `ChainID.AURORA` items from here?

That'd maintain xchain functionality if we want to do any testing or anything (as we've done in the past)

Reply:
makes sense. Will add